app/actions/app_feature_update.rb

```diff
@@ -1,15 +1,41 @@
 module VCAP::CloudController
   class AppFeatureUpdate
-    def self.update(feature_name, app, message)
-      case feature_name
-      when AppFeaturesController::SSH_FEATURE
-        app.update(enable_ssh: message.enabled)
-      when AppFeaturesController::REVISIONS_FEATURE
-        app.update(revisions_enabled: message.enabled)
-      when AppFeaturesController::SERVICE_BINDING_K8S_FEATURE
-        app.update(service_binding_k8s_enabled: message.enabled)
-      when AppFeaturesController::FILE_BASED_VCAP_SERVICES_FEATURE
-        app.update(file_based_vcap_services_enabled: message.enabled)
+    class InvalidCombination < StandardError; end
+
+    class << self
+      def update(feature_name, app, app_feature_update_message)
+        app.update({ database_column(feature_name) => app_feature_update_message.enabled })
+      end
+
+      def bulk_update(app, manifest_features_update_message)
+        flags = {}
+
+        manifest_features_update_message.features&.each do |feature_name, enabled|
+          flags[database_column(feature_name)] = enabled
+        end
+
+        return if flags.empty?
+
+        check_invalid_combination!(app, flags)
+        app.update(flags)
+      end
+
+      private
+
+      def database_column(feature_name)
```

Comment:
The name of this method confused me a bit. Could we rename it to something like `feature_column_name` or `database_column_name`?